py/desisim/targets.py

```diff
@@ -236,6 +236,9 @@ def get_targets(nspec, program, tileid=None, seed=None, specmin=0):
     Options:
         tileid: (int) tileid, used for setting RA,dec
         seed: (int) random number seed
+        specify_targets: dict;  Define target properties like magnitude and redshift
+                                see simspec.templates.specify_galparams_dict() or 
+                                simsepc.templates.specify_starparams_dict()
```

Comment:
Sigh; I think sphinx doesn't like this kind of multi-line syntax.  if you make each item a bullet list I think it will work, e.g.
```
Options:
    * tileid: ...
    * seed: ...
    * specify_targets: ...
                    see simspec...
    * specmin: ...
```

`python setup.py build_sphinx` can pre-test your docs before pushing, but note that you'll need sphinx 1.5 not latest 1.6.2.

Please try that out and holler for help if you can't get it working.